src/config.ts

```diff
@@ -36,6 +36,19 @@ const config = {
       clientId: 'ca5c4c520da868387c52',
     },
   },
+  environment: 'testnet',
+  seedNodesTest: {
+    v359vgrgmqf1r5g4fvisiddjknjko6bmm4qv7646jr7fi9enbfuug: {
```

Comment:
These are dummy values atm. We haven't actually created the actual seed nodes yet. This can only be done in #194.

Therefore under `testnet`, this would be random string for now.

For `mainnet`, it can be empty, since we don't have it yet.